### PR TITLE
[DOC] Convert Agent doc to Alloy

### PR DESCRIPTION
**Pull request by knylander-grafana**
Converts content from using Agent to Alloy. 

Changes: 
* Most of the Agent configuration sections are removed. 
* Links are updated to point to the Alloy docs 
* Global search to check for any references to Agent

What needs to be checked: 
* Verify that the config file snippets work. The configuration file changes were made with help from @clayton-cornell (Alloy writer).
* Check any graphics to see if they mention Agent or Alloy, and update if needed. 
* Sampling scrap targets
* Helm chart doc updates

Fixes https://github.com/grafana/pyroscope/issues/3271

### CODE REVIEW

File: docs/sources/configure-client/grafana-agent/_index.md [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
 
+## Legacy collector, Grafana Agent
+
+{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="next" >}}
+
+Grafana Agent is a legacy tool for collecting and forwarding profiling data.
+Agent supports for eBPF and Golang in pull mode.
+For information about Agent, refer to [Grafana Agent Flow](https://grafana.com/docs/agent/<AGENT_VERSION>/flow/).
+
+Instructions for using Grafana Agent is available in documentation for Pyroscope v1.8 and earlier.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/_index.md:37]:
```suggestion
Instructions for using Grafana Agent are available in documentation for Pyroscope v1.8 and earlier.
```

File: docs/sources/_index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ cards:
       description: Learn how to install and configure Grafana Pyroscope with several examples.
     - title: Instrument your app and configure the client
       href: ./configure-client/
-      description: When sending profiles to Pyroscope, you can choose between SDK instrumentation and auto-instrumentation using the Grafana Agent. This document explains these two techniques and guide you when to choose each one.
+      description: When sending profiles to Pyroscope, you can choose between SDK instrumentation and auto-instrumentation using Grafana Alloy. This document explains these two techniques and guide you when to choose each one.
```
Comment (knylander-grafana) [inline docs/sources/_index.md:33]:
```suggestion
      description: When sending profiles to Pyroscope, you can choose between SDK instrumentation and auto-instrumentation using Grafana Alloy. This document explains these two techniques and helps you choose one.
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-linux.md [hunk 3]
```diff
@@ -42,51 +40,41 @@ uname -r
...
-Verify that the agent is correctly installed by running:
+Follow the [installation instructions](https://grafana.com/docs/alloy/<ALLOY_VERSION>/set-up/install/linux/) to download and install Alloy for your current Linux distribution.
 
-```shell
-grafana-agent-flow --version
-```
+## Configure Grafana Alloy
 
-## Configure the Grafana Agent
+To configure the Alloy eBPF profiler to profile local processes, you'll need to set the `targets_only` flag to `false` and add a default target in the `pyroscope.ebpf` component.
```
Comment (korniltsev) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-linux.md:49]: `targets_only` is an undocumented flag and was used as a workaround before `discovery.process` was available.

now they need to use `discovery.process` https://grafana.com/docs/pyroscope/latest/configure-client/grafana-agent/ebpf/setup-linux/

File: docs/sources/configure-client/grafana-agent/java/_index.md [hunk 4]
```diff
@@ -75,37 +71,32 @@ see [profiler-options](https://github.com/async-profiler/async-profiler?tab=read
 
 ### Set privileges for the collector
 
-You must run the collector, either Grafana Alloy (recommended) or Agent (legacy), as root and inside host `pid` namespace for the `pyroscope.java`
-and `discover.process` components to work.
+You must run the collector, either Grafana Alloy as root and inside host `pid` namespace for the `pyroscope.java` and `discover.process` components to work.
```
Comment (korniltsev) [inline docs/sources/configure-client/grafana-agent/java/_index.md:74]: > either Grafana Alloy as root 

something is off here, we don't need  "either"

File: docs/sources/configure-client/language-sdks/nodejs.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ Pyroscope.start()
 
 [comment]: <> (TODO This needs its own page like https://grafana.com/docs/pyroscope/latest/configure-client/grafana-agent/go_pull/)
 {{< admonition type="note" >}}
-If you'd prefer, you can use Pull mode using [Grafana Alloy](https://grafana.com/docs/alloy/latest/) (recommended) or [Grafana Agent]({{< relref "../grafana-agent" >}}) (legacy).
+If you'd prefer, you can use Pull mode using [Grafana Alloy](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-agent/).
```
Comment (korniltsev) [inline docs/sources/configure-client/language-sdks/nodejs.md:52]: should we link to the pull mod page directly ? https://grafana.com/docs/pyroscope/latest/configure-client/grafana-agent/go_pull/

**Review by korniltsev — APPROVED** [review]
lgtm

File: docs/sources/configure-client/grafana-agent/_index.md [hunk 6]
```diff
@@ -1,33 +1,41 @@
...
 
-# Grafana Alloy and Grafana Agent
+# Grafana Alloy
 
 You can send data from your application using Grafana Alloy (preferred) or Grafana Agent (legacy) collectors.
 Both collectors support profiling with eBPF, Java, and Golang in pull mode.
 
-[Grafana Alloy](https://grafana.com/docs/alloy/latest/) is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
+[Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_VERSION>/) is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
 Alloy uniquely combines the very best OSS observability signals in the community.
```
Comment (marcsanmi) [inline docs/sources/configure-client/grafana-agent/_index.md:16]: One minor suggestion: should we consider standardizing the usage of `Grafana Alloy` vs. `Alloy` throughout the docs for consistency?

**Review by marcsanmi — APPROVED** [review]
LGTM 👍

File: docs/sources/configure-client/_index.md [hunk 7]
```diff
@@ -57,24 +57,22 @@ Here's how to use Pyroscope SDKs:
...
-{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="next" >}}
-
-You can use Grafana Agent for auto-instrumentation or the Pyroscope instrumentation SDKs.
+You can use Grafana Alloy for auto-instrumentation or the Pyroscope instrumentation SDKs.
 The method you choose depends on your specific use case and requirements.
 
 Here are some factors to consider when making the choice:
 
-- Ease of setup: The Grafana Agent is an ideal choice for a quick and straightforward setup without modifying your application's code. Note that eBPF profiling supports some languages (for example, Golang) better than others. More robust support for Python, Java, and other languages are in development.
+- Ease of setup: The Grafana Alloy is an ideal choice for a quick and straightforward setup without modifying your application's code. eBPF profiling supports some languages (for example, Golang) better than others. More robust support for Python, Java, and other languages are in development.
```
Comment (aleks-p) [inline docs/sources/configure-client/_index.md:67]:
```suggestion
- Ease of setup: Grafana Alloy is an ideal choice for a quick and straightforward setup without modifying your application's code. eBPF profiling supports some languages (for example, Golang, Python) better than others. More robust support for Java and other languages is in development.
```

File: docs/sources/configure-client/grafana-agent/_index.md [hunk 8]
```diff
@@ -1,33 +1,41 @@
 ---
-title: "Grafana Alloy and Grafana Agent"
-menuTitle: "Grafana Alloy and Grafana Agent"
-description: "Send data from your application via using Grafana Alloy or Grafana Agent."
+title: "Grafana Alloy"
+menuTitle: "Grafana Alloy"
+description: "Send data from your application via using Grafana Alloy."
```
Comment (aleks-p) [inline docs/sources/configure-client/grafana-agent/_index.md:4]:
```suggestion
description: "Send data from your application using Grafana Alloy."
```

File: docs/sources/configure-client/grafana-agent/ebpf/_index.md [hunk 9]
```diff
@@ -38,15 +38,12 @@ Python is the only supported high-level language, as long as `python_enabled=tru
 Other high-level languages like Java, Ruby, PHP, and JavaScript require additional work to show stack traces of methods in these languages correctly.
 Currently, the CPU usage for these languages is reported as belonging to the runtime's methods.
 
+## eBPF using Grafana Alloy
 
-## eBPF via the Grafana Agent
+The Grafana Alloy is a lightweight, all-in-one collector that can collect, transform, and ship observability data.
```
Comment (aleks-p) [inline docs/sources/configure-client/grafana-agent/ebpf/_index.md:43]:
```suggestion
Grafana Alloy is a lightweight, all-in-one collector that can collect, transform, and ship observability data.
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md [hunk 10]
```diff
@@ -100,29 +98,39 @@ controller:
...
 If you're using your own Pyroscope server, you can remove the `basic_auth` section altogether.
 {{% /admonition %}}
 
-For more information, refer to the [Configure the Grafana Pyroscope data source documentation](/docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/configure-pyroscope-data-source/).
 
-## Install the Grafana Agent
+## Install the Grafana Alloy
 
-To install the Grafana Agent, run:
+To install the Grafana Alloy, run:
```
Comment (aleks-p) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md:120]:
```suggestion
To install Grafana Alloy, run:
```

**Review by aleks-p — APPROVED** [review]
Looks good to me, added a few minor suggestions.

File: docs/sources/configure-client/grafana-agent/_index.md (hunk 6 above)
Comment (clayton-cornell) [reply docs/sources/configure-client/grafana-agent/_index.md:16]: Within the Alloy docs we use a variable to manage this... and the first mention of Alloy uses `Grafana Alloy` and all subsequent mentions within a single topic are just  `Alloy`

File: docs/sources/configure-client/grafana-agent/_index.md [hunk 11]
```diff
@@ -1,33 +1,41 @@
 ---
-title: "Grafana Alloy and Grafana Agent"
```
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/configure-client/grafana-agent/_index.md:2]: Will do!

File: docs/sources/configure-client/grafana-agent/_index.md [hunk 12]
```diff
@@ -1,33 +1,41 @@
...
+# Grafana Alloy
 
 You can send data from your application using Grafana Alloy (preferred) or Grafana Agent (legacy) collectors.
 Both collectors support profiling with eBPF, Java, and Golang in pull mode.
 
-[Grafana Alloy](https://grafana.com/docs/alloy/latest/) is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
+[Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_VERSION>/) is a vendor-neutral distribution of the OpenTelemetry (OTel) Collector.
 Alloy uniquely combines the very best OSS observability signals in the community.
-Grafana Alloy uses configuration file written using River.
+Grafana Alloy uses configuration files written in Alloy configuration syntax.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/_index.md:17]:
```suggestion 
Alloy uses configuration files written in Alloy configuration syntax.
```

File: docs/sources/configure-client/grafana-agent/_index.md (hunk 6 above)
Comment (knylander-grafana) [reply docs/sources/configure-client/grafana-agent/_index.md:16]: I should check through these docs and make sure we're consistent.

File: docs/sources/configure-client/grafana-agent/_index.md [hunk 13]
```diff
@@ -61,18 +71,18 @@ In pull mode, the collector periodically retrieves profiles from Golang applicat
...
-1. Ensure the collector is set to pull mode and targeting the correct pprof endpoints. For step-by-step instructions, visit the [Go (Pull Mode)](/docs/pyroscope/latest/configure-client/grafana-agent/go_pull) documentation.
+1. Install and configure Grafana Alloy on the same machine or container where your application runs.
+1. Ensure Alloy is set to pull mode and targeting the correct pprof endpoints. For step-by-step instructions, visit the [Go (Pull Mode)](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-agent/go_pull) documentation.
 1. The collector queries the pprof endpoints of your Golang application, collects the profiles, and forwards them to the Pyroscope server.
 
 ## Next steps
 
-Whether using eBPF for versatile system and application profiling or relying on Golang's built-in pprof endpoints in pull mode, Grafana Agent and Grafana Alloy collectors offer streamlined processes to gather essential profiling data.
+Whether using eBPF for versatile system and application profiling or relying on Golang's built-in pprof endpoints in pull mode, Grafana Alloy collectors offer streamlined processes to gather essential profiling data.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/_index.md:85]:
```suggestion
Whether using eBPF for versatile system and application profiling or relying on Golang's built-in pprof endpoints in pull mode, Alloy collectors offer streamlined processes to gather essential profiling data.
```

File: docs/sources/configure-client/grafana-agent/ebpf/_index.md [hunk 14]
```diff
@@ -38,15 +38,12 @@ Python is the only supported high-level language, as long as `python_enabled=tru
...
+For profiling, Grafana Alloy can be configured to collect eBPF profiles and send them to Pyroscope.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/_index.md:44]:
```suggestion
For profiling, Alloy can be configured to collect eBPF profiles and send them to Pyroscope.
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md [hunk 15]
```diff
@@ -1,36 +1,36 @@
...
 # Setup eBPF Profiling on Docker
 
-To set up eBPF profiling with Grafana Agent on Linux, you need to:
+To set up eBPF profiling with Grafana Alloy on Linux, you need to:
 
 - Verify that your system meets the requirements.
-- Create a Grafana Agent configuration file. For more information, refer to the [Configuration reference][config-reference].
-- Run Grafana Agent or Grafana Alloy.
+- Create a Grafana Alloy configuration file. For more information, refer to the [Configuration reference][config-reference].
+- Run Grafana Alloy.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md:14]:
```suggestion
- Run Alloy.
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md [hunk 16]
```diff
@@ -1,36 +1,36 @@
...
 
 # Setup eBPF Profiling on Docker
 
-To set up eBPF profiling with Grafana Agent on Linux, you need to:
+To set up eBPF profiling with Grafana Alloy on Linux, you need to:
 
 - Verify that your system meets the requirements.
-- Create a Grafana Agent configuration file. For more information, refer to the [Configuration reference][config-reference].
-- Run Grafana Agent or Grafana Alloy.
+- Create a Grafana Alloy configuration file. For more information, refer to the [Configuration reference][config-reference].
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md:13]:
```suggestion
- Create an Alloy configuration file. For more information, refer to the [Configuration reference][config-reference].
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md [hunk 17]
```diff
@@ -40,16 +40,18 @@ To print the kernel version of your docker host, run:
...
 
-Make sure you have a kernel version >= 4.9.
+The kernel version must be 4.9 or later.
 
-## Configure Grafana Agent
+## Configure Grafana Alloy
 
-You can configure the Grafana Agent eBPF profiler to profile local containers.
-To do so, use the `discovery.docker` component to discover local containers and the `pyroscope.ebpf` component to profile them.
+You can configure Grafana Alloy eBPF profiler to profile local containers.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md:47]:
```suggestion
You can configure Alloy eBPF profiler to profile local containers.
```

File: docs/sources/configure-client/grafana-agent/java/_index.md (hunk 4 above)
Comment (clayton-cornell) [reply docs/sources/configure-client/grafana-agent/java/_index.md:74]: Also running Alloy as root is the default expectation Running as non-root is a special case (there are docs for setting up non-root)

File: docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md [hunk 18]
```diff
@@ -75,37 +76,41 @@ pyroscope.write "endpoint" {
...
 If you're using your own Pyroscope server, you can remove the `basic_auth` section altogether.
 {{% /admonition %}}
 
-For more information, refer to the [Configure the Grafana Pyroscope data source documentation](/docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/configure-pyroscope-data-source/).
 
-## Start Grafana Agent
+## Start Grafana Alloy
 
-To start Grafana Agent with Docker, run:
+To start Grafana Alloy with Docker, run:
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md:94]:
```suggestion
To start Alloy with Docker, run:
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md [hunk 19]
```diff
@@ -1,32 +1,30 @@
...
 
-To set up eBPF profiling with Grafana Agent on Kubernetes, you need to:
+To set up eBPF profiling with Grafana Alloy on Kubernetes, you need to:
 
 - Verify that your cluster meets the prerequisites.
 - Add the Grafana helm repository.
-- Create a Grafana Agent configuration file. For more information, refer to [Configuration reference][config-reference].
-- Install Grafana Agent.
+- Create a Grafana Alloy configuration file. For more information, refer to [Configuration reference][config-reference].
+- Install Grafana Alloy, refer to the [installation instructions](https://grafana.com/docs/alloy/<ALLOY_VERSION>/set-up/install/kubernetes/)
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md:15]:
```suggestion
- Create an Alloy configuration file. For more information, refer to [Configuration reference][config-reference].
- Install Alloy, refer to the [installation instructions](https://grafana.com/docs/alloy/<ALLOY_VERSION>/set-up/install/kubernetes/)
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md [hunk 20]
```diff
@@ -55,18 +54,17 @@ helm repo update
...
 ```shell
-helm search repo grafana/grafana-agent
+helm search repo grafana/alloy
 ```
 
-It should return a list of available versions of Grafana Agent.
+The command returns a list of available versions of Grafana Alloy.
 
-## Create a Grafana Agent configuration file
+## Create a Grafana Alloy configuration file
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md:62]:
```suggestion
## Create an Alloy configuration file
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md [hunk 21]
```diff
@@ -100,29 +98,39 @@ controller:
...
+For more information, refer to the [Configure the Grafana Pyroscope data source documentation](/docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/configure-pyroscope-data-source/).
 
 {{% admonition type="note" %}}
 If you're using your own Pyroscope server, you can remove the `basic_auth` section altogether.
 {{% /admonition %}}
 
-For more information, refer to the [Configure the Grafana Pyroscope data source documentation](/docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/configure-pyroscope-data-source/).
 
-## Install the Grafana Agent
+## Install the Grafana Alloy
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-kubernetes.md:118]:
```suggestion
## Install Alloy
```

File: docs/sources/configure-client/grafana-agent/ebpf/setup-linux.md [hunk 22]
```diff
@@ -1,31 +1,29 @@
...
-To set up eBPF profiling with Grafana Agent on Linux, you need to:
+To set up eBPF profiling with Grafana Alloy on Linux, you need to:
 
 - Verify that your system meets the requirements.
-- Install the Grafana Agent Flow mode.
-- Create a Grafana Agent configuration file. For more information, refer to [Configuration reference][config-reference].
-- Run Grafana Agent or Grafana Alloy.
+- Install [Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_VERSION>/set-up/install/linux/).
+- Create a [Grafana Alloy configuration file](https://grafana.com/docs/alloy/<ALLOY_VERSION>/configure/linux/). For more information, refer to [Configuration reference](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/).
+- Run Grafana Alloy.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-linux.md:15]:
```suggestion
- Install [Grafana Alloy](https://grafana.com/docs/alloy/<ALLOY_VERSION>/set-up/install/linux/).
- Create an [Alloy configuration file](https://grafana.com/docs/alloy/<ALLOY_VERSION>/configure/linux/). For more information, refer to [Configuration reference](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/).
- Run Alloy.
```

File: docs/sources/configure-client/grafana-agent/go_pull.md [hunk 23]
```diff
@@ -34,27 +34,21 @@ Ensure your Golang application exposes pprof endpoints.
...
+You can use the sample Alloy collector configuration file to send data to Pyroscope.
 
 To install Alloy, refer to [Grafana Alloy installation](https://grafana.com/docs/alloy/<ALLOY_VERSION>/get-started/install/).
 
-{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="next" >}}
-
 ### Prepare the collector configuration file
 
-In the Grafana Alloy or Grafana Agent Flow configuration file, you need to add at least two blocks: `pyroscope.write` and `pyroscope.scrape`.
+In the Grafana Alloy configuration file, you need to add at least two blocks: `pyroscope.write` and `pyroscope.scrape`.
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/go_pull.md:47]:
```suggestion
In the Alloy configuration file, you need to add at least two blocks: `pyroscope.write` and `pyroscope.scrape`.
```

File: docs/sources/configure-client/grafana-agent/go_pull.md [hunk 24]
```diff
@@ -115,14 +109,9 @@ In the Grafana Alloy or Grafana Agent Flow configuration file, you need to add a
     docker run -p 4040:4040 grafana/pyroscope
     ```
 
-1. Start the collector:
+1. To start Grafana Alloy v1.2 and later: Replace `configuration.alloy` with your configuration filename: <br> `alloy run configuration.alloy`
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/go_pull.md:112]:
```suggestion
1. To start Alloy v1.2 and later: Replace `configuration.alloy` with your configuration filename: <br> `alloy run configuration.alloy`
```

File: docs/sources/configure-client/grafana-agent/java/_index.md (hunk 4 above)
Comment (knylander-grafana) [reply docs/sources/configure-client/grafana-agent/java/_index.md:74]: Good catch.

File: docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md [hunk 25]
```diff
@@ -75,37 +76,41 @@ pyroscope.write "endpoint" {
...
   --privileged \
   -p 12345:12345 \
-  grafana/agent:latest \
-    run --server.http.listen-addr=0.0.0.0:12345 /etc/agent/config.river
+  grafana/alloy:latest \
+    run --server.http.listen-addr=0.0.0.0:12345 /etc/alloy/alloy.config
 ```
 
-> Note: The `--pid=host` and `--privileged` flags are required to profile local containers with ebpf.
+{{< admonition type="note >}}
```
Comment (knylander-grafana) [inline docs/sources/configure-client/grafana-agent/ebpf/setup-docker.md:107]:
```suggestion
{{< admonition type="note" >}}
```

File: docs/sources/configure-client/language-sdks/nodejs.md (hunk 5 above)
Comment (knylander-grafana) [reply docs/sources/configure-client/language-sdks/nodejs.md]: DOne.

File: docs/sources/configure-client/grafana-agent/_index.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/sources/configure-client/grafana-agent/_index.md:2]: Done. And updated all links within the doc set from grafana-agent to grafana-alloy.

File: docs/sources/configure-client/grafana-agent/_index.md (hunk 6 above)
Comment (knylander-grafana) [reply docs/sources/configure-client/grafana-agent/_index.md:17]: Done.

File: docs/sources/upgrade-guide/_index.md [hunk 26]
```diff
@@ -64,7 +64,9 @@ We stripped the pyroscope CLI of all subcommands that were related to the client
 * `pyroscope connect`
 * `pyroscope agent`
 
-This strategic shift has been in the works for some time as we transition away from CLI-based profiling towards embracing native integrations. Moving forward, we encourage users to take advantage of native integrations tailored to specific programming languages, such as pip packages for Python, .NET packages for .NET applications, Ruby gems for Ruby applications, and so on. Our [eBPF integration]({{< relref "../configure-client/grafana-agent/ebpf" >}}) is also a good way to get profiling data for your whole cluster.
+This strategic shift has been in the works for some time as we transition away from CLI-based profiling towards embracing native integrations.
+Moving forward, we encourage users to take advantage of native integrations tailored to specific programming languages, such as pip packages for Python, .NET packages for .NET applications, Ruby gems for Ruby applications, and so on.
+The [eBPF integration](../configure-client/grafana-alloy/ebpf) is also a good way to get profiling data for your whole cluster.
```
Comment (knylander-grafana) [inline docs/sources/upgrade-guide/_index.md:69]:
```suggestion
The [eBPF integration](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-alloy/ebpf) is also a good way to get profiling data for your whole cluster.
```

File: docs/sources/upgrade-guide/_index.md [hunk 27]
```diff
@@ -64,7 +64,9 @@ We stripped the pyroscope CLI of all subcommands that were related to the client
 * `pyroscope connect`
 * `pyroscope agent`
 
-This strategic shift has been in the works for some time as we transition away from CLI-based profiling towards embracing native integrations. Moving forward, we encourage users to take advantage of native integrations tailored to specific programming languages, such as pip packages for Python, .NET packages for .NET applications, Ruby gems for Ruby applications, and so on. Our [eBPF integration]({{< relref "../configure-client/grafana-agent/ebpf" >}}) is also a good way to get profiling data for your whole cluster.
+This strategic shift has been in the works for some time as we transition away from CLI-based profiling towards embracing native integrations.
+Moving forward, we encourage users to take advantage of native integrations tailored to specific programming languages, such as pip packages for Python, .NET packages for .NET applications, Ruby gems for Ruby applications, and so on.
+The [eBPF integration](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-alloy/ebpf) is also a good way to get profiling data for your whole cluster.
```
Comment (knylander-grafana) [inline docs/sources/upgrade-guide/_index.md:69]:
```suggestion
The [eBPF integration](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/configure-client/grafana-alloy/ebpf/) is also a good way to get profiling data for your whole cluster.
```

**Comment by knylander-grafana** [comment]
Checking if it's safe to backport this PR to 1.9. It changes a lot of pages.

**Pull request merged**